lib/phoenix/sync/write.ex

```diff
@@ -0,0 +1,970 @@
+defmodule Phoenix.Sync.Write do
```

Comment:
Naming wise, I would lean to `Writes` over `Write`. However, you call the controller `MutationsController`, so perhaps this is `Phoenix.Sync.Mutations`?

Reply:
`Mutations` was what I was leaning to